client/webserver/site/src/css/market_dark.scss

```diff
@@ -17,31 +17,11 @@ body.dark {
       background-color: #141488;
     }
 
-    .markettab.selected {
-      background-color: #fff1;
-    }
-
     .market-bal {
       border: 1px solid $dark_border_color;
       border-radius: 3px;
     }
 
-    .brdrleft {
-      border-left: 1px solid $dark_border_color;
-    }
-
-    .brdrright {
-      border-right: 1px solid $dark_border_color;
-    }
-
-    .brdrtop {
-      border-top: 1px solid $dark_border_color;
-    }
-
-    .brdrbottom {
-      border-bottom: 1px solid $dark_border_color;
-    }
```

Comment:
All these seems to have already been defined in `main_dark.scss`, as I understood it, after removing these - the ones in `main` will be used.

Reply:
Maybe they were set because [these didn't work](https://github.com/decred/dcrdex/pull/2003#discussion_r1053494915) properly ?